apps/consulting/posts/openai-pseudocode.md

```diff
@@ -0,0 +1,276 @@
+---
+title: 'Exploring the Impact of Feedback Loops on LLM Code Generation'
+published: March 27, 2023
+author: chris-ostrouchov
+description: 'As an experiment we wanted to add a feedback loop into LLM generated code via type annotations, docstrings, automated tests, and user feedback. We developed a simple package "pseudoscode" which uses the OpenAI API for generating code.'
```

Comment:
@bskinn @Katebrack  Should the text use "we" or "I"?  @costrouc is the only author, so I think it should be "I".

Reply:
@ericdatakelly I see your point but I do like 'we' to imply a team. It could be he's the author but several worked on the experiment. IMO, "I" puts a lot of emphasis on the one person. Thoughts @bskinn ?

Reply:
Since this is a company blog, not a personal one, my inclination is to prefer 'we' in just about every circumstance.

I'm finding it hard to think of a situation where a post would reach the point of publication on the blog, in which it was important to distinguish the voice of an author from that of the company/team.